### PR TITLE
Update doc for Autoscaping : default true

### PR DESCRIPTION
**Pull request by forresst**
Fixes : #511

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -693,17 +693,18 @@ follow a few rules when writing asynchronous templates:
...
-recommended that you do this for security reasons.
+By default, nunjucks will escape all output. It's recommended
+that you do this for security reasons. If you turn off autoescaping,
+nunjucks will render all output as it is by default.
 
-Autoescaping is rather simplistic in nunjucks right now. All you have
-to do is pass the `autoescape` option as `true` to the `Environment`
-object. In the future, you will have more control over which files
-this kicks in.
+Autoescaping is rather simplistic in nunjucks right now. In the future,
```
Comment (carljm) [inline docs/api.md]: If we're updating this doc anyway, let's just remove the sentence about "in the future..." No need to make promises that nobody (AFAIK) currently has plans to implement.

Comment (carljm) [inline docs/api.md]: ...so let's just get rid of this whole paragraph.

File: docs/api.md [hunk 2]
```diff
@@ -693,17 +693,18 @@ follow a few rules when writing asynchronous templates:
...
+you will have more control over which files this kicks in.
+
+To desactivate it, all you have to do is pass the `autoescape` option as
```
Comment (carljm) [inline docs/api.md]: s/desactivate/deactivate/

File: docs/api.md (hunk 1 above)
Comment (forresst) [inline docs/api.md]: OK i'll do !

File: docs/api.md (hunk 2 above)
Comment (forresst) [inline docs/api.md]: Ooops !!!!

**Comment by carljm** [comment]
Thanks! Merged.

**Comment by forresst** [comment]
@carljm Done !
As a reminder, it will provide an update in gh-pages.

**Comment by carljm** [comment]
Thanks for the reminder, I pushed an update to the live docs.

**Pull request merged**